question-servers/elements/pl_answer_panel.py

```diff
@@ -18,5 +18,8 @@ def parse(element_html, element_index, data):
 def grade(element_html, element_index, data):
     return data
 
-def testcase(element_html, element_index, data):
+def test(element_html, element_index, data):
     return data
+
+def file(element_html, element_index, data):
```

Comment:
Thinking about it further I guess I agree with @nwalters512 that we should allow element functions to be missing. In particular, if we require all element functions to be present and people write their own course-specific elements, then if we add a new element function it will cause everyone's existing elements to be broken, which seems bad.